### PR TITLE
OBSDOCS-99 - Logging components by version

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.12+

Issue: [OBSDOCS-99](https://issues.redhat.com//browse/OBSDOCS-99) 

Link to docs preview: https://73212--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/cluster-logging#logging-components-by-version_cluster-logging

QE review: @anpingli  / @QiaolingTang 
- [x] QE has approved this change.

Additional information:
SME: @alanconway  / @jcantrill

### CODE REVIEW

**Comment by libander** [comment]
@anpingli / @kabirbhartiRH  - /lgtm please.

**Comment by periklis** [comment]
LGTM

File: modules/logging-components-by-version.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+[id="logging-components-by-version_{context}"]
+= Logging components by version
+
+For components with a version value of `master` or `main`, the current version is the latest commit to the respective branch.
+
+[options="header"]
+.Component versions
+|===
+| Release | `eventrouter` | `logfilemetricexplorer` | `loki`   | `lokistack-gateway` | `opa-openshift` | `vector`
+| 5.9     | master   | v5.9                  | v2.9.4 | main           | main       | v0.33.0
```
Comment (kabirbhartiRH) [inline modules/logging-components-by-version.adoc:13]: I think the vector version for Logging 5.9 is v0.34.1

File: modules/logging-components-by-version.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+| 5.8     | master   | v5.8                  | v2.9.4 | main           | main       | v0.28.1
+| 5.7     | master   | v5.7                  | v2.9.4 | main           | main       | v0.28.1
```
Comment (kabirbhartiRH) [inline modules/logging-components-by-version.adoc:15]: `logfilemetricexplorer` branch looks to be on 'main' for Logging 5.7 too

File: modules/logging-components-by-version.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+For components with a version value of `master` or `main`, the current version is the latest commit to the respective branch.
+
+[options="header"]
+.Component versions
+|===
+| Release | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+| 5.9     | master   | main                  | v2.9.4 | main           | main       | v0.34.0
+| 5.8     | master   | v5.8                  | v2.9.4 | main           | main       | v0.28.1
+| 5.7     | master   | main                  | v2.9.4 | main           | main       | v0.28.1
+| 5.6     | master   | v1.1                  | v2.9.4 | main           | main       | v0.20.0
```
Comment (kabirbhartiRH) [inline modules/logging-components-by-version.adoc:16]: `logfilemetricexplorer` version should be `main` for Logging 5.6 too

File: modules/logging-components-by-version.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+[id="logging-components-by-version_{context}"]
+= Logging components by version
+
+For components with a version value of `master` or `main`, the current version is the latest commit to the respective branch.
+
+[options="header"]
+.Component versions
+|===
+| Release | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+| 5.9     | master   | main                  | v2.9.4 | main           | main       | v0.34.0
```
Comment (kabirbhartiRH) [inline modules/logging-components-by-version.adoc:13]: small correction, vector version for Logging 5.9 should be `v0.34.1`

**Comment by kabirbhartiRH** [comment]
@libander Could you please get the PR reviewed by Collection SME.
cc. @cahartma

**Comment by libander** [comment]
> @libander Could you please get the PR reviewed by Collection SME. cc. @cahartma

@jcantrill ?

File: modules/logging-components-by-version.adoc [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+For components with a version value of `master` or `main`, the current version is the latest commit to the respective branch.
+
+[options="header"]
+.Component versions
+|===
+| Release | `eventrouter` | `logfilemetricexplorer` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+| 5.9     | master   | main                  | v2.9.4 | main           | main       | v0.34.1
+| 5.8     | master   | v5.8                  | v2.9.4 | main           | main       | v0.28.1
+| 5.7     | master   | main                  | v2.9.4 | main           | main       | v0.28.1
+| 5.6     | master   | main                  | v2.9.4 | main           | main       | v0.21.0
```
Comment (periklis) [inline modules/logging-components-by-version.adoc:16]: Loki is on all supported version at v3.1.0 and with the next z-streams (5.8.13 on Sept 26th, 5.9.7+5.6.24 on Oct 2nd) on 3.1.1. 5.7 is not supported anymore but it was lastly released with v2.9.8

Comment (anpingli) [reply modules/logging-components-by-version.adoc:16]: How can we make this page updated as per each z-stream release.

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.18` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.17. And any PR going into `main` must also target the latest version branch (enterprise-4.18).

If the update in your PR does NOT apply to version 4.18 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: modules/logging-components-by-version.adoc (hunk 5 above)
Comment (periklis) [reply modules/logging-components-by-version.adoc:16]: We are already on v3.2.0 with Loki here :wave:

**Pull request closed without merging**